compiler/damlc/lib/DA/Cli/Damlc/Packaging.hs

```diff
@@ -223,6 +221,15 @@ createProjectPackageDb projectRoot opts thisSdkVer deps dataDeps = do
             mbPkgVersion
             deps
             dependencies
+  where
+    dbPath = projectPackageDatabase </> lfVersionString (optDamlLfVersion opts)
+    clearPackageDb = do
+        -- Since we reinitialize the whole package db during `daml init` anyway,
```

Comment:
Should say `daml damlc init` instead of `daml init`.

`daml damlc init` should really be `daml damlc init-package-db` or so.